### ISSUE TITLE
Large download fails with "Failed - Network error" message

### ISSUE THREAD

**Issue by nilayp** [Backblaze-B2-Samples/multithreaded-downloader-js#1]
If I try to download a really large file, say 50GB, sometimes it works. 

<img width="2219" alt="multi_threaded_downloader_and_downloads" src="https://user-images.githubusercontent.com/744246/41553754-926cd1d6-72e7-11e8-989e-b947f9ae08a4.png">

Other times, I get "Failed - Network error" messages from the browser. However, the download continues... it just doesn't get assembled in my downloads folder. (It is somewhere on my computer, because my total disk space does decrease.)

<img width="696" alt="downloads" src="https://user-images.githubusercontent.com/744246/41553806-b737a25c-72e7-11e8-8039-c694f491ec40.png">

If I try to resume, I get the following error:

<img width="693" alt="downloads" src="https://user-images.githubusercontent.com/744246/41553819-c0da79f6-72e7-11e8-8e9a-9e2699246125.png">

**Comment by chpmnrssll** [comment]
Yep, the individual chunk download/retry works all the way to 100% but the file stops being written kind of at random. I thought it might be some kind of timeout for taking too long
writing to the stream, but couldn't pinpoint it. 

The service worker limit is a big roadblock, but there has to be some way of creating a writeable stream to a file without using service worker. I'm still scouring the web for some info/hack to get around it and am still pretty determined to find a way to make this work.

**Comment by nilayp** [comment]
Out of curiosity, do you know where the partial files end up? I ran a bunch of 50GB failed downloads today and my computer is down 150GB of storage... But I can't find the partial downloads anywhere. :)

Looking forward to seeing a workaround for the writeable stream to file.

**Comment by chpmnrssll** [comment]
I have no idea on that one, maybe hidden in some chrome system folder? When it fails here (on Arch linux) I still get a partial file in my Downloads directory.

I've been digging around at Mega.nz, they use several different methods but none of them requires a service worker. Lots of good info here: https://github.com/meganz/webclient

**Comment by jimmaay** [comment]
trying to download a 700GB File using b2 3.4.0 download-file-by-id and it keeps failing around 500-700 GB's. I've retried multiple times already. Any way to get it to download fully or at least resume if it fails so I don't have to retry all the way from the beginning?
